WordPress/Classes/ViewRelated/Blog/CreateNewBlog/CreateNewBlogViewController.m

```diff
@@ -0,0 +1,551 @@
+#import "CreateNewBlogViewController.h"
+#import "WordPressComApi.h"
+#import "WPNUXMainButton.h"
+#import "WPNUXSecondaryButton.h"
+#import "WPWalkthroughTextField.h"
+#import "WPAsyncBlockOperation.h"
+#import "WPComLanguages.h"
+#import "WPWalkthroughOverlayView.h"
+#import "WPNUXUtility.h"
+#import "WPStyleGuide.h"
+#import "UILabel+SuggestSize.h"
+#import "WPAccount.h"
+#import "Blog.h"
+#import "WordPressComServiceRemote.h"
+#import "AccountService.h"
+#import "BlogService.h"
+#import "ContextManager.h"
+#import "NSString+XMLExtensions.h"
+#import "Constants.h"
+
+#import "WordPress-Swift.h"
+
+@interface CreateNewBlogViewController ()<UITextFieldDelegate,UIGestureRecognizerDelegate> {
```

Comment:
@kwonye these should be updated and declared each as a `@property` instead of as ivars within the private extension.